ansible_collections/arista/avd/roles/eos_designs/templates/network_services/vlan-interfaces.j2

```diff
@@ -8,26 +8,36 @@ vlan_interfaces:
 {%             if svi.profile is arista.avd.defined %}
 {%                 set svi_profile = svi_profiles[svi.profile] | arista.avd.default() %}
 {%             endif %}
+{%             if svi_profile.parent_profile is arista.avd.defined %}
+{%                 set parent_profile = svi_profiles[svi_profile.parent_profile] | arista.avd.default({}) %}
```

Comment:
The parent_profile is only implemented partially in the template below.
Instead of having to add it on every variable, I suggest to do a combine similar to what we have under `connected_endpoints`.
It will require some more refactoring in this template, so let me know if you want assistance.
Example from connected_endpoints:
```jinja
{%             set adapter_settings = {} | combine(parent_profile | arista.avd.default({}),
                                                   adapter_profile | arista.avd.default({}),
                                                   adapter, recursive=true, list_merge='replace') %}
```

Reply:
Yeah - I thought not every nod would required the parent profile but that was foolish. I will make a commit to fix this and have every node be capable of the parent profile. I will attach another commit to this PR trying to refactor the template.

Reply:
Refactoring this was actually easier then expected - I hope this meets the expectations.